exec/java-exec/src/main/java/org/apache/drill/exec/store/dfs/WorkspaceSchemaFactory.java

```diff
@@ -104,6 +107,9 @@ public WorkspaceSchemaFactory(DrillConfig drillConfig, FileSystemPlugin plugin,
       final FormatMatcher fallbackMatcher = new BasicFormatMatcher(formatPlugin,
           ImmutableList.of(Pattern.compile(".*")), ImmutableList.<MagicString>of());
       fileMatchers.add(fallbackMatcher);
+      dropFileMatchers = fileMatchers.subList(0, fileMatchers.size() - 1);
```

Comment:
Why would we have different matchers for drop file.  Shouldn't we simply share the behavior across all matching behaviors?